xarray/core/variable.py

```diff
@@ -1150,7 +1199,9 @@ def to_numpy(self) -> np.ndarray:
         data = self.data
 
         # TODO first attempt to call .to_numpy() once some libraries implement it
-        if isinstance(data, dask_array_type):
+        # cubed has to be imported dynamically as cubed imports rechunker which imports xarray
```

Comment:
Is this fixable upstream in rechunker?

Reply:
Fixed in https://github.com/pangeo-data/rechunker/pull/123

Reply:
Thanks!